make/get_avbin.ps1

```diff
@@ -0,0 +1,37 @@
+$AVBIN_32_URL = "https://staff.washington.edu/larsoner/avbin.dll"
```

Comment:
I don't get why this variable is getting set, since it appears to never be accessed, and the full URLs are provided again inside the `DownloadAVBin` function.  Other than that LGTM.

Reply:
Cruft!